ui/src/main/webapp/src/app/executions/executions-layout.component.ts

```diff
@@ -117,6 +117,16 @@ export class ExecutionsLayoutComponent extends ProjectLayoutComponent implements
                 this.execution,
                 'migration-issues'
             ),
+            new ContextMenuItem(
+                'Static Reports',
+                'fa-external-link',
+                true,
+                WindupExecutionService.formatStaticReportUrl(this.execution),
+                null,
+                null,
+                '_blank',
+                true
+            )
```

Comment:
Fixed... I changed the icon and made it only visible if static reports are available.